superset-frontend/src/components/CopyToClipboard/index.jsx

```diff
@@ -102,7 +102,7 @@ class CopyToClipboard extends React.Component {
 
   renderLink() {
     return (
-      <span>
+      <span style={{ display: 'inline-flex', alignItems: 'center' }}>
```

Comment:
I'm not a huge fan of inline styles if there's a better place to stick this.

Reply:
... at the very least, we could set up the `css={theme => css...` boilerplate to make it look like real CSS with theme support.

Reply:
Changed to use the `css` property instead of `style`.